content/hi/docs/tutorials/configuration/configure-redis-using-configmap.md

```diff
@@ -0,0 +1,253 @@
+---
+reviewers:
+- eparis
+- pmorie
+title: कॉन्फ़िगमैप का उपयोग करके रेडिस को कॉन्फ़िगर करना
+content_type: ट्यूटोरियल
+---
+
+<!-- overview -->
+
+यह पृष्ठ कॉन्फिग मैप का उपयोग करके रेडिस को कॉन्फ़िगर करने का वास्तविक विश्व उदाहरण प्रदान करता है और [कॉन्फिग मैप का उपयोग करने के लिए पॉड को कॉन्फ़िगर करें] (/docs/tasks/configure-pod-container/configure-pod-configmap/) कार्य पर आधारित है।
+
+
+
+## {{% heading "objectives" %}}
+
+
+* रेडिस कॉन्फ़िगरेशन वैल्यू के साथ एक कॉन्फ़िगमैप बनाएं
+* एक रेडिस पॉड बनाएं जो बनाए गए कॉन्फ़िगमैप को माउंट और उपयोग करता है
+* जांच करें कि कॉन्फ़िगरेशन सही ढंग से लागू किया गया था
+
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+* इस पृष्ठ पर दिखाया गया उदाहरण `kubectl` 1.14 और उससे ऊपर के संस्करण के साथ काम करता है।
+* समझें [Configure a Pod to Use a ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/).
+
+
+
+<!-- lessoncontent -->
+
+
+## वास्तविक विश्व उदाहरण: कॉन्फिग मैप का उपयोग करके रेडिस को कॉन्फ़िगर करना
+
+कॉन्फिगरेशन मैप में स्टोर्ड डेटा का उपयोग करके रेडिस कैश को कॉन्फ़िगर करने के लिए नीचे दिए गए चरणों का पालन करें।
+
+सबसे पहले एक खाली कॉन्फ़िगरेशन ब्लॉक के साथ एक कॉन्फिग मैप बनाएं:
+
+```shell
+cat <<EOF >./example-redis-config.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: example-redis-config
+data:
+  redis-config: ""
+EOF
+```
+
+रेडिस पॉड मेनिफेस्ट के साथ ऊपर बनाए गए कॉन्फ़िगमैप को लागू करें:
+
+```shell
+kubectl apply -f example-redis-config.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/website/main/content/en/examples/pods/config/redis-pod.yaml
+```
+
+रेडिस पॉड मेनिफेस्ट की कंटेंट की जांच करें और निम्नलिखित पर ध्यान दें:
+
+* `config` नामक वॉल्यूम `spec.volumes[1]` द्वारा बनाया गया है
+* `spec.volumes[1].configMap.items[0]` के अंतर्गत `कुंजी` और `पथ` `redis-config` कुंजी को उजागर करता है
+  `example-redis-config` `config` वॉल्यूम पर `redis.conf` नामक फ़ाइल के रूप में कॉन्फिग मैप।
+* `config` वॉल्यूम को `spec.containers[0].volumeMounts[1]` द्वारा `/redis-master` पर माउंट किया जाता है।
+
+इसका `example-redis-config` से `data.redis-config` में डेटा को उजागर करने का शुद्ध प्रभाव है
+पॉड के अंदर `/redis-master/redis.conf` के रूप में ऊपर कॉन्फिग मैप।
+
+{{% code_sample file="pods/config/redis-pod.yaml" %}}
```

Comment:
It seems the PR build preview is encountering [failures](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/65ac20a44ff61900082a5583) as the `{{ code }}` shortcode is unable to locate the `redis-pod.yaml` for Hindi localization.

_Build Errors:_
> 7:37:57 PM: ERROR [hi] "redis-pod.yaml" not found in "docs/tutorials/configuration/pods/config"
> 7:37:57 PM: ERROR [hi] "example-redis-config.yaml" not found in "docs/tutorials/configuration/pods/config"
> 7:38:00 PM: Failed during stage "building site": Build script returned non-zero exit code: 2


To address this, I suggest copying the yaml file(s) under [content/en/examples/pods/config/redis-pod.yaml](https://github.com/kubernetes/website/tree/main/content/en/examples/pods/config) to a similar location for Hindi by creating the yaml file(s) under **`content/en/examples/pods/config/`**. This could likely resolve the error.

Reply:
Thanks, @dipesh-rawat, but no example folder is in the ( content/hi ). So first of all we need to create an example folder under content/hi to create yaml file. 
![git](https://github.com/kubernetes/website/assets/124805747/e9972693-e0b3-43c2-a98e-f611fb23c42c)

Reply:
As part of this pull request, please create all the required folder and example files for Hindi language that mimic the structure of the [English content](https://github.com/kubernetes/website/tree/main/content/en/examples/pods/config), in order to complete the localization of this page.